packages/api/src/index.js

```diff
@@ -23,54 +23,80 @@ const router = Router()
 router.options('*', corsOptions)
 router.all('*', envAll)
 
+router.get('*', withMode(READ_ONLY))
+router.head('*', withMode(READ_ONLY))
+router.post('*', withMode(READ_WRITE))
+router.delete('*', withMode(READ_WRITE))
+
+/**
+ * Takes any number of functions and invokes them all one after the other
+ *
+ * @param  {...function(...any):any} fns
+ * @returns
+ */
+const compose = (...fns) => x => fns.reduceRight((y, f) => f(y), x)
```

Comment:
minor: extract out to a utils/fn.js or similar. It's a one liner, but it distracts from the main purpose of this file – to init the thing.

Reply:
Good idea!